frontend/src/components/pages/topics/Tab.Messages/index.tsx

```diff
@@ -672,10 +672,24 @@ class SaveMessagesDialog extends Component<{ messages: TopicMessage[] | null, on
                     <ModalHeader>{title}</ModalHeader>
                     <ModalBody>
                         <div>Select the format in which you want to save {count == 1 ? 'the message' : 'all messages'}</div>
-                        <Radio.Group value={this.format} onChange={e => this.format = e.target.value}>
-                            <Radio value="json" style={this.radioStyle}>JSON</Radio>
-                            <Radio value="csv" disabled={true} style={this.radioStyle}>CSV</Radio>
-                        </Radio.Group>
+                        <Box py={2}>
+                            <RadioGroup
+                                name="format"
+                                value={this.format}
+                                onChange={value => this.format = value}
+                                options={[
+                                    {
+                                        value: 'json',
+                                        label: 'JSON'
+                                    },
+                                    {
+                                        value: 'csv',
+                                        label: 'CSV',
+                                        disabled: true
+                                    }
```

Comment:
I know it was like this before, but is there a point in having an always disabled CSV radio group option? @weeco maybe you know why

Reply:
I know, I just didn't want to introduce any "breaking" changes just based on my decision. I try to do this migration 1:1

Reply:
Leave it in for now.
We might have a solution to the CSV issue (that is, doing it the "simple" way, where only the root properties are different columns, and the payload/content stays as json).
Depends on what people actually expect from saving as CSV.
So overall, its not completely off the table that we might want to add it in the future if people want it.